cli/pipeline/run.go

```diff
@@ -145,6 +145,11 @@ func runPipelineStep(pipeline *PipelineDefinition, step *PipelineDefinitionStep,
 		return err
 	}
 
+	var gracePeriod int64
+	opts := metav1.DeleteOptions{}
+	opts.GracePeriodSeconds = &gracePeriod
+	pods.Delete(podDefinition.PodName, &opts)
```

Comment:
We already kill the pod [here](https://github.com/deliveroo/paddle/pull/21/files#diff-601c4a0058a4ea3c41605e612eeea9f5R135): what does this add?

Reply:
@me - correc.t We have been seeing quite a few occurences over the last 2 weeks or so where one of our pods gets stuck in a `Terminating` phase at times for even 2 days. By forcing a kill with `grace_period = 0` this forces kubernetes to instantly kill the pod. 

@jteh found this issue: https://github.com/docker/for-linux/issues/1 which seems to be the root cause of the problem, however we can't update the version of the docker client b/c of `kops` does not support that yet.

Reply:
I see! Should we add the `GracePeriodSeconds` option [here](https://github.com/deliveroo/paddle/blob/a1e847e53311c1bd757700d6be848bfd2667c310/cli/pipeline/run.go#L227) instead?

Reply:
Right - i totally missed that was getting called before my change. I'll add the opt in there :+1: